src/goose.rs

```diff
@@ -2426,30 +2470,32 @@ mod tests {
 
         // Make a GET request to the mock http server and confirm we get a 200 response.
         assert_eq!(mock_index.times_called(), 0);
-        let goose = user.get("/").await;
-        let status = goose.response.unwrap().status();
-        assert_eq!(status, 200);
+        if let Ok(goose) = user.get("/").await {
```

Comment:
I would not do that as a panic here is correct if that fails.

Reply:
Because if that fails, this part of the test is not run and that's definitely wrong ;)

Reply:
Yeah, silly of me. It actually would still catch the failure as below this {} block it verifies it actually invoked the mock endpoint, but agreed it's better to just panic here.

Reply:
fixed